README.adoc

```diff
@@ -86,83 +86,45 @@ include::{common-includes}/gitclone.adoc[]
 == Creating the Dockerfile
 
 A Dockerfile is a collection of instructions to build a Docker image that you can then
-run as a container. Every Dockerfile begins
-with a parent or base image on top of which these instructions are run. For example, you can start
-your image from scratch and execute commands that download Java, or you can start from
-an image that already contains a Java installation.
-
-Use the `openliberty/open-liberty:javaee7` image that contains the Open Liberty
-runtime as the parent image.
-
-Before you write your Dockerfile, familiarize yourself with the types of instructions that are compatible
-with Dockerfiles. Use the following instructions to build a Dockerfile for this guide:
-
-[cols="70, 100", options="header"]
-|===
-| *Command* | *Description*
-| FROM <image> | This command initializes a new build stage and indicates the parent image from which your image
-is built, or you can use `FROM scratch` to indicate that your image is a base image.
-| RUN <commands> | This command executes the commands in a new layer on top of the current image.
-| CMD ["executable", "param1", "param2"] | This command defines the command to be executed when the image runs.
-| ENTRYPOINT ["executable", "param1", "param2"] | This command works similarly to the `CMD` command, but unlike
-`CMD`, only the `--entrypoint` flag can override `ENTRYPOINT`.
-|===
-
-For a more extensive and detailed list, see the official https://docs.docker.com/engine/reference/builder[Docker documentation].
-
-=== Writing your Dockerfile
-
-To begin, specify `openliberty/open-liberty:javaee7` as the parent image. This image contains the Open Liberty runtime.
+run as a container. Every Dockerfile begins with a parent or base image on top of which these instructions
+are executed. For example, you can start your image from scratch and execute commands that download Java,
+or you can start from an image that already contains a Java installation.
 
 Create a regular `start/Dockerfile` file:
 
 [source, indent=0]
 ----
-include::finish/Dockerfile[tags=from]
+include::finish/Dockerfile[tags=**]
 ----
 
-By default, the Open Liberty runtime is downloaded to the `/opt/ol/wlp` directory.
+The `FROM` instruction initializes a new build stage and indicates the parent image from which your
+image is built. If you don't need a parent image, then use `FROM scratch`. This makes your image
+a base image. In this case, you're using the `openliberty/open-liberty:javaee7` image are your parent
+image, which comes with the latest Open Liberty runtime.
```

Comment:
I think we want "**as** your parent image", correct?

If you want more sentence variety (since we use "which" in the previous two sentences), you could say, "...parent image. This image comes with the latest..." or you can leave it as it is with "which."

Reply:
Good catch. I'll keep "which" since I dont think its too bad.